lib/game_loop.ex

```diff
@@ -0,0 +1,52 @@
+import GetUserMove
+import BestComputerSpot
+import WinningCombo
+
+defmodule GameLoop do
+
+  def _yes_no_checker(command_prompt)do
```

Comment:
Any reason this function isn't defined inside the ElixirTicTacToe module? Looks like that's the only place where it's used.

Reply:
Resolved in rebase